core/EAP7-1266_WFCORE-4314_enchance_keystore_commands_alias.adoc

```diff
@@ -0,0 +1,343 @@
+= Enchance read-alias and read-aliases keystore commands
+:author:            Bartosz Baranowski
+:email:             bbaranow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+ Elytron keystore expose `read` commands to access content:
+ * read-alias
+ * read-aliases
+
+ There are two problems with those commands. The `read-alias` command is too verbose by default. When alias is displayed after read command, output include hex format of public-key and encoded fields. This is usually not something required if one inspects key and it clouds output with dump of unreadable string of hex digits.
+ Second problem is that read-aliases is in fact "read-aliases-names". In order to read content of keystore one has to read alias names with `read-aliases` and issue `read-alias` for each name. To make it more user friendly, simple changes are required:
+ * VERBOSE - new parameter for both commands. Default value would be `false`. This parameter control inclusion of bulk hex fields of key(public-key and encoded)
+ * RECURSIVE - new parameter for `read-aliases`. Default value `false`. If set otherwise, `read-aliases` will read recurse into keys, displaying more than names.
+
+CAUTION: default value of VERBOSE parameter, in current implementation is `true`. Explicity definition of parameter with default value of `false` is a change in behavior.
```

Comment:
Instead of having `verbose` get set to false by default, I think it should actually default to true. That way, the operations will continue to return the same values they currently return today by default, in case anyone is relying on that behaviour.

Reply:
Indeed. This is why caution meta is there. I did think about backward compatibility, however in this case impact could be negligible? Im fine with inverting this.

Reply:
For backwards compatibility, please update this so that ```verbose``` is set to ```true``` by default.

Reply:
Did, Im just waiting for test everything before I push